vignettes/EIRprevmatch.Rmd

```diff
@@ -15,68 +15,152 @@ knitr::opts_chunk$set(
+
+## *Pf*PR~2-10~ matching using malariaEquilibrium
+
+The `malariaEquilibrium` package function `human_equilibrium()` can be used to return the canonical equilibrium solution of the system for a given EIR and the matching *Pf*PR~2-10~ calculated from the output (see <https://github.com/mrc-ide/malariaEquilibrium> for more information). First, we generate a large vector of EIR values for which we want to generate matching *Pf*PR~2-10~ values. Next, we load the package's default parameter set (`Jamie_parameters.rds`) and, for each EIR value generated, run the `human_equilibrium()` function. The `human_equilibrium()` function returns a dataframe containing the proportion of each age class in each state variables at equilibrium. The *Pf*PR~2-10~ is then calculated from the output for each EIR value by summing the proportion of people aged 2-10 with cases of malaria (`pos_M`) to get the proportion of individuals aged 2-10 with malaria, and dividing this proportion by the proportion of the population between the ages 2-10 (`prop`). Finally, we store the matching EIR and *Pf*PR~2-10~ values in a data frame.
+
+```{r}
+
+# Establish a range of EIR values to generate matching PfPR2-10 values for
+malEq_EIR <- seq(from = 0.1, to = 50, by = 0.5)
+
+# Load the base malariaSimulation parameter set:
+eq_simparams <- malariaEquilibrium::load_parameter_set("Jamie_parameters.rds")
+
+# Use human_equilibrium to calculate the PfPR2-10 values for the range of
+# EIR values
+malEq_prev <- vapply(
+  malEq_EIR,
+  function(eir) {
+    eq <- malariaEquilibrium::human_equilibrium(
+      eir,
+      ft = 0,
+      p = eq_simparams,
+      age = 0:100
+    )
+    sum(eq$states[3:11, 'pos_M']) / sum(eq$states[3:11, 'prop'])
+  },
+  numeric(1)
+)
+
+# Establish a dataframe containing the matching EIR and PfPR2-10 values
+malEq_P2E <- cbind.data.frame(EIR = malEq_EIR, prev = malEq_prev)
+
+```
+
+This method does not involve running simulations and can therefore be used to return matching *Pf*PR~2-10~ for a wide range of EIR values very quickly. However, it is only viable for systems at steady state and with a fixed set of parameter values and with limited options for capturing the effects of interventions designed to combat malaria transmission. Often, when running `malariasimulation` simulations, one or both of these conditions are not met and, therefore, a different solution is required.
```

Comment:
I don't think you can actually include any interventions with this method

Reply:
@pwinskill - does malariaEquilibrium allow users to specify only a limited set of interventions (presumably with static values through time), or no interventions at all? I couldn't work it out from the GitHub page/README

Reply:
So malariaEquilibrium allows users to specify a proportion of the population that receive effective treatment (ft in the inputs), so I've changed the text to explain this.